docs/index.html

```diff
@@ -0,0 +1,175 @@
+    />
+    <link rel="stylesheet" href="./css/main.css" />
+    <title>Ibhicheckesho | Home</title>
+  </head>
+  <body>
+    <header id="header-home">
+      <ul class="social-links-list">
+        <li class="social-li">
+          <a class="social-a" href="#twitter"
+            ><i class="fab fa-facebook-square"></i
+          ></a>
+        </li>
+        <li class="social-li">
+          <a class="social-a" href="https://twitter.com/MnembaChambuya"
+            ><i class="fab fa-twitter"></i
+          ></a>
+        </li>
+        <li class="social-li"><a class="social-a" href="#logout">Logout</a></li>
+        <li class="social-li">
+          <a class="social-a" href="#mypage">My Page</a>
+        </li>
+        <li class="social-li">
+          <a class="social-a" href="#tanzanian">Tanzanian</a>
+        </li>
+      </ul>
+      <div class="container">
+        <nav id="main-nav">
+          <div class="container">
+            <div id="logo">
+              <a href="index.html"
+                ><img src="./images/logo.png" alt="Watubaki-logo"
+              /></a>
+            </div>
+            <div class="baga-icon-container">
+              <input id="baga-toggler" type="checkbox" />
+              <div class="baga-icon">
+                <div></div>
+              </div>
+            </div>
+            <ul class="nav-menu">
+              <li><a class="nav-menu-link" href="./index.html">Home</a></li>
+              <li><a class="nav-menu-link" href="./about.html">About</a></li>
+            </ul>
+          </div>
+        </nav>
+        <section id="header-content">
+          <h1>
+            <span id="header-content-span">Hello! W'anna Lough? Welcome</span
+            ><span id="header-content-span1">Watubaki</span
+            ><span id="header-content-span2">Standup Comedy</span
+            ><span id="header-content-span3">March 2022</span>
+          </h1>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum
+            quibusdam similique tenetur obcaecati maxime, molestias
+            exercitationem. Eaque voluptatum optio, tempora sit recusandae error
+            tenetur minus ipsam laboriosam? Ea, alias. Pariatur?
+          </p>
+          <hgroup>
+            <h2>2022.10.15(THU) ~ 16(FRI)</h2>
+            <h3>@Mlimani City Shopping Center.</h3>
+          </hgroup>
+        </section>
+      </div>
+    </header>
+    <main id="home-main">
+      <section id="schedule" class="main-section">
+        <div class="container">
+          <h2>Schedule <span></span></h2>
+          <div class="cards">
+            <section class="card">
+              <i class="fas fa-anchor"></i>
+              <h3>Opening</h3>
+              <p class="shedule-p">
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quo
+                sed sapiente, voluptatibus labore fugit repellat?
+              </p>
```

Comment:
- [ ] It would be better if you add the background, to make the design closer to the original. Here I found you an image. Hope it will help you to fix the issue.
`https://raw.githubusercontent.com/elmar8287/capstone/main/images/bg-main-program.PNG`